zeebe/util/src/main/java/io/camunda/zeebe/util/TlsConfigUtil.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.X509Certificate;
+import java.util.Arrays;
+
+public class TlsConfigUtil {
```

Comment:
:wrench: Just a heads up, since utility classes aren't meant to be instantiated, you would typically also make it final, and provide a private default constructor. This makes the intent of the class clearer (i.e. just a collection of utility methods).

Reply:
done